services/core-api/app/api/verifiable_credentials/manager.py

```diff
@@ -81,8 +83,8 @@ class W3CCred(BaseModel):
     credentialSchema: List[dict]
 
 
-def convert_date_to_iso_datetime(date: date) -> str:
-    return datetime(date.year, date.month, date.day, 0, 0, 0, tzinfo=ZoneInfo("UTC")).isoformat()
+def convert_date_to_iso_datetime(dt: datetime) -> str:
+    return datetime(dt.year, dt.month, dt.day, 0, 0, 0, tzinfo=ZoneInfo("UTC")).isoformat()
```

Comment:
this function I think I might like to see in a generic helper/utils/whatever file. I think we might have more use for it!

Reply:
It's currently converting it to a string that is needed specifically for vcdm. so maybe it's name should be clearer.